src/Appwrite/Utopia/Response/Model.php

```diff
@@ -8,7 +8,7 @@ abstract class Model
 {
     const TYPE_STRING = 'string';
     const TYPE_INTEGER = 'integer';
-    const TYPE_FLOAT = 'float';
+    const TYPE_FLOAT = 'double';
```

Comment:
I still think we should use float here to avoid coupling ourself with legacy PHP behavior. We can probably cast our own `gettype` method that behaves correctly. Let me know what you think.

Reply:
I don't think it's a good idea to wrap a method in the standard library - I think it'd be hard to maintain and onboard.